flexiblepricing/models.py

```diff
@@ -46,6 +51,28 @@ def __str__(self):
         )
 
 
+class FlexiblePriceTier(TimestampedModel):
```

Comment:
Can you add a `__str__` method here so that it's easier to distinguish between individual objects?